### DISCUSSION TITLE
Is there any way to give workflow resubmit custom name?

### DISCUSSION THREAD

**Discussion by Rachel-Rabinowitz** [argoproj/argo-workflows#9222] [Q&A] [answered]
I have to resubmit workflow  with `argo resubmit wf_name`
and because it creates a new run I want to give the new run a custom name
I tried the `--name` flag but it is unknown
I also tried `metadata.name=custom-name` but it refers to the workflow I resubmit and not the new one
Any other option to do that?
thanks

**Comment by terrytangyuan** [comment, marked as answer]
It's not supported yet but might be a good feature request.

**Comment by Rachel-Rabinowitz** [comment]
Thanks
